components/server/src/workspace/messagebus-integration.ts

```diff
@@ -117,6 +118,7 @@ export class MessageBusIntegration extends AbstractMessageBusIntegration {
         const listener = new HeadlessWorkspaceLogListener(this.messageBusHelper, callback, workspaceID);
         const cancellationTokenSource = new CancellationTokenSource()
         this.listen(listener, cancellationTokenSource.token);
+        increaseMessagebusTopicReads(listener.topic.name)
         return Disposable.create(() => cancellationTokenSource.cancel())
     }
 
```

Comment:
Why not listen in `listenForPrebuildUpdatableQueue` as well? :thinking:

Reply:
PrebuildUpdatableQueue's topic didn't have a `name` attribute. I had a quick talk with @csweichel last week and we agreed on removing the metric over there.

@csweichel I feel like we had another reason that I can't remember right now. Do you remember if there were more reasons to not expose a metric for this queue?

Reply:
primarily the fact that it's a queue and not an exchange and we didn't find a good naming scheme.